### ISSUE TITLE
Feature Request: Support For Markdown Tables

### ISSUE THREAD

**Issue by ghost** [bachya/Sifttter-Redux#9]
I guess this would be part of multi-line support but it would be great to be able to put enter data in a table format:

http://bentsai.wordpress.com/2014/02/26/day-one-templates-using-launch-center-pro/

(Using the name of the text file as the table title).

**Comment by bachya** [comment]
Thanks, @mralexhay! Right now, as you know, all entries are simply bullets; would you see this as being a new type of addition to a journal entry (i.e., rather than processing everything as a bullet, process the Markdown in it as-is)?

(hopefully that makes sense. :) )

**Comment by ghost** [comment]
Yeah, I suppose a large part of the power and flexibility of Sifttter is the simplicity in the single format of bullet pointed lines but perhaps one of way of implementing tables would be as an additional data entry type, i.e anything between the usual date line and the @done would be parsed into day one and ordered in the same way etc.

#### Example raw IFTTT entry:

```
- February 22, 2014 at 19:29PM

### weight
|  |  | 
| ------------- |:-------------:|
| Weight (kg)       | 99 | 
| BMI      | 22      | |
| Fat mass (%) | 15     | 

@done
```

#### Example Day One formatting:

![screen shot 2014-02-28 at 19 13 58](https://f.cloud.github.com/assets/4391511/2297433/8b047a00-a0ac-11e3-8dbb-db8d35783397.png)

Not sure if other people have much of a use-case for this or not but it's a nice, clean way of displaying any fixed set of numbers that relate to the day as a whole rather than at a specific time. 

(I just stuck those in manually obviously so they're not alphabetical!)

**Comment by bachya** [comment]
I like that a lot. Maybe what I'll do is flex the expected file format of Sifttter files a bit to allow for any Markdown to exist.

**Comment by ghost** [comment]
Yeah, that would be perfect!

**Comment by bachya** [comment]
@mralexhay -- thanks for your patience! I think we're getting close to having something ready. Would you be willing to be a beta tester? :)

**Comment by ghost** [comment]
Absolutely!—

**Comment by bachya** [comment]
Great! I'll keep you posted. :)

**Comment by bachya** [comment]
@mralexhay -- just emailed you instructions. Thanks!
